### PR TITLE
Raise exception on missing artifact file

### PR DESCRIPTION
**Pull request by keilr**
`artifact` module:

- added "File not found." exception

### CODE REVIEW

File: plugins/modules/artifact.py [hunk 1]
```diff
@@ -91,6 +91,8 @@ def main():
 
         sha256 = module.params["sha256"]
         if module.params["file"]:
+            if not os.path.exists(module.params["file"]):
+                raise Exception("File not found.")
```
Comment (mdellweg) [inline plugins/modules/artifact.py]: Can we raise a SqueezerException here. There is no need to show the stacktrace, as we already know what's going wrong.

Comment (keilr) [reply plugins/modules/artifact.py]: Thanks for the hint, I wasn't aware of the custom exception.

Comment (mdellweg) [reply plugins/modules/artifact.py]: np It's handled here: https://github.com/pulp/squeezer/blob/develop/plugins/module_utils/pulp.py#L79

**Comment by mdellweg** [comment]
Can you squash your commits into one before we merge?

**Comment by keilr** [comment]
> Can you squash your commits into one before we merge?

done

**Review by mdellweg — APPROVED** [review]

**Comment by mdellweg** [comment]
I'm convinced, the ci failure is unrelated.

fyi @gerrod
https://github.com/pulp/squeezer/pull/72/checks?check_run_id=3775351157#step:9:1029
https://github.com/pulp/squeezer/pull/72/checks?check_run_id=3775351157#step:9:1068
https://github.com/pulp/squeezer/pull/72/checks?check_run_id=3775351157#step:9:1144

**Comment by mdellweg** [comment]
Thank you for this contribution!

**Comment by gerrod** [comment]
@mdellweg - thanks for the mention, but not entirely sure how you'd like me to use this information... suspect you may have tagged the wrong developer 🤪

**Comment by mdellweg** [comment]
> @mdellweg - thanks for the mention, but not entirely sure how you'd like me to use this information... suspect you may have tagged the wrong developer zany_face

@gerrod Right. Sorry!

**Pull request merged**